src/screens/LocalConfirmNotificationCodeScreen.tsx

```diff
@@ -39,11 +39,8 @@ class LocalConfirmNotificationCodeScreen extends PureComponent<Props, State> {
     numberAttempt: 0,
   };
 
-  componentWillUnmount() {
-    this.props.setError('');
-  }
-
   setCode = (userCode: string) => {
+    this.props.setError('');
```

Comment:
I would do 
`this.props.error && this.props.setError('');`
so we won't call action when error is already reseted